orchagent/bulker.h

```diff
@@ -11,6 +11,8 @@
 #include "logger.h"
 #include "sai_serialize.h"
 
+#define MAX_BULK_SIZE 1000
```

Comment:
can we make this configurable? maybe via cmd line option or configuration file?

Reply:
Added a command-line option to configure the max bulk size.